fortran/test/fetch_content_integration/test_get_micm_version.F90

```diff
@@ -0,0 +1,8 @@
+program test_get_micm_version
+    use musica_util, only: string_t
+    use musica_micm, only: get_micm_version
+    implicit none
+    type(string_t) :: micm_version
+    micm_version = get_micm_version()
+    print *, "MICM version ", micm_version%get_char_array()
+end program test_get_micm_version
```

Comment:
Looks great! I would just now pull the content of this test directly into the tutorial so they don't get out-of-sync. Similar to this:
https://github.com/NCAR/micm/blob/ee5c03e904d9dc3d60619dc9dbce06232bd32852/docs/source/user_guide/rate_constant_tutorial.rst?plain=1#L46-L49

Also, I would move this to a `test/tutorial/` folder or something similar